### PR TITLE
fix: replaced `appc ` command with `ti `

### PR DESCRIPTION
**Pull request by caspahouzer**

### CODE REVIEW

File: docs/guide/Titanium_SDK/Titanium_SDK_FAQ.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ An example of updated `tiapp.xml` file might look something like this:
 
 To verify your buildtools version, go to your Android SDK location: `/users/<user>/Library/Android/sdk/build-tools/ <version_number>`
 
-You can change the build tools version using this command: `appc ti config android.buildTools.selectedVersion ##.##.##`
+You can change the build tools version using this command: `ti ti config android.buildTools.selectedVersion ##.##.##`
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_FAQ.md]: `ti ti`

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Platform_SDKs/Installing_the_Windows_SDK.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ The minimum requirements (builds tools and SDKs, not including Windows Mobile Em
 
 ## Troubleshooting
 
-* Appcelerator Studio does not show any emulators or connected devices/`appc ti info -t windows` does not display any information
+* Appcelerator Studio does not show any emulators or connected devices/`ti ti info -t windows` does not display any information
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Platform_SDKs/Installing_the_Windows_SDK.md]: same `ti ti`

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Titanium_SDK_Continuous_Builds.md [hunk 3]
```diff
@@ -31,13 +31,13 @@ If you discover a corrupt package, simply download each previous day's release u
...
-* Be sure to log in to the CLI if you are not already logged in (`appc login`)
+* Be sure to log in to the CLI if you are not already logged in (`ti login`)
 
-* Run `appc ti sdk install -b master`
+* Run `ti ti sdk install -b master`
 
 * The command above will install the latest CI build from `master` branch. You may substitute branch names seen at [http://builds.appcelerator.com](http://builds.appcelerator.com) to grab bugfix/maintenance CI builds. These are typically of the naming scheme `\d+_\d+_X,`i.e.`7_0_X`
 
-* You can view additional command line options by executing:`appc ti sdk install -h`
+* You can view additional command line options by executing:`ti ti sdk install -h`
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Titanium_SDK_Continuous_Builds.md]: `ti ti`

**Comment by m1ga** [comment]
ok, there are quiet a few `ti ti` commands :smile: check your search and replace :wink: 
Try to search and replace `ti ti` with `ti` now. Think that will remove those duplicates and I don't have to mark all

File: docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_9.x/Titanium_SDK_9.2.0_RC_Release_Note.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ This initial beta mostly focuses on enabling the target and necessary build chan
 
 * Currently there are no new APIs exposed for macOS development such as Menu APIs. As a result apps will feel more like an iPad app currently.
 
-* Appcelerator Studio and our editor plugins currently do not expose macOS as a build target, currently the only way to build is to using the CLI via `[appc run|ti build] -p ios -T macos`
+* Appcelerator Studio and our editor plugins currently do not expose macOS as a build target, currently the only way to build is to using the CLI via `[ti build|ti build] -p ios -T macos`
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_9.x/Titanium_SDK_9.2.0_RC_Release_Note.md]: that should just be  `ti build -p ios -T macos`

File: docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_9.x/Titanium_SDK_9.2.0_GA_Release_Note.md [hunk 5]
```diff
@@ -33,7 +33,7 @@ This initial beta mostly focuses on enabling the target and necessary build chan
 
 * Currently there are no new APIs exposed for macOS development such as Menu APIs. As a result apps will feel more like an iPad app currently.
 
-* Appcelerator Studio and our editor plugins currently do not expose macOS as a build target, currently the only way to build is to using the CLI via `[appc run|ti build] -p ios -T macos`
+* Appcelerator Studio and our editor plugins currently do not expose macOS as a build target, currently the only way to build is to using the CLI via `[ti build|ti build] -p ios -T macos`
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_9.x/Titanium_SDK_9.2.0_GA_Release_Note.md]: that should just be  `ti build -p ios -T macos`

File: docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_8.x/Titanium_SDK_8.0.2.GA_Release_Note.md [hunk 6]
```diff
@@ -45,7 +45,7 @@ As of this GA release, the previous Titanium SDK patch release is no longer supp
 
 * [TIMOB-27101](https://jira.appcelerator.org/browse/TIMOB-27101) - Android: Window open() wrongly does a shared-element fade-in animation by default as of 8.0.1
 
-* [TIMOB-27102](https://jira.appcelerator.org/browse/TIMOB-27102) - iOS: "appc run" wrongly edits a Titanium SDK installed file causing studio to beachball and rebuild its workspace
+* [TIMOB-27102](https://jira.appcelerator.org/browse/TIMOB-27102) - iOS: "ti build" wrongly edits a Titanium SDK installed file causing studio to beachball and rebuild its workspace
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_8.x/Titanium_SDK_8.0.2.GA_Release_Note.md:48]: you'll get the same merge conflicts as in the other ticket with those lines :wink: It has a jira link that will be changed in the other PR

Comment (caspahouzer) [reply docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_8.x/Titanium_SDK_8.0.2.GA_Release_Note.md:48]: but I already merged the master. Or do I've to merge the Jira PR and solve conflicts then?

File: docs/guide/Titanium_SDK/Titanium_SDK_Guide/Titanium_Command-Line_Interface_Reference/Titanium_CLI_Plugins.md [hunk 7]
```diff
@@ -886,7 +886,7 @@ warn(String message): void
 
 ### nodeappc
 
-The node-appc class provides access to various utility libraries.
+The node-ti class provides access to various utility libraries.
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Guide/Titanium_Command-Line_Interface_Reference/Titanium_CLI_Plugins.md]: that is still called `node-appc`

File: docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# Transfer your app from Appcelerator CLI to Titanium CLI
+# Transfer your app from Titanium CLI to Titanium CLI
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md]: should still be "Appcelerator" here

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Titanium_SDK_Continuous_Builds.md [hunk 9]
```diff
@@ -27,17 +27,17 @@ If you discover a corrupt package, simply download each previous day's release u
 
 ## Installing the build
 
-### Appcelerator CLI install
+### Titanium CLI install
 
 To install a Titanium SDK using the `appc` cli, follow these steps:
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Installation_and_Configuration/Installing_Titanium_SDK_Continuous_Builds.md]: `ti` cli

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Prerequisites/Installing_Node.md [hunk 10]
```diff
@@ -5,7 +5,7 @@ weight: '20'
 
 # Installing Node
 
-Node.js is required for several Appcelerator components, including the Axway Appcelerator CLI, Alloy, and the API Builder. Node.js is also required to build the Titanium SDK.
+Node.js is required for several Appcelerator components, including the Axway Titanium CLI, Alloy, and the API Builder. Node.js is also required to build the Titanium SDK.
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Prerequisites/Installing_Node.md]: remove `Axway`

File: docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Titanium_Platform_Overview.md [hunk 11]
```diff
@@ -17,7 +17,7 @@ At a high-level, Titanium is the combination of:
 
 * [Titanium SDK](#titanium-sdk)
 
-* [Appcelerator CLI](#appcelerator-cli)
+* [Titanium CLI](#appcelerator-cli)
```
Comment (m1ga) [inline docs/guide/Titanium_SDK/Titanium_SDK_Getting_Started/Titanium_Platform_Overview.md]: link target should be `#titanium-cli`

**Review by m1ga — APPROVED** [review]
LGTM

**Comment by caspahouzer** [comment]
@cb1kenobi i would be a honor to me if you would merge it together

**Comment by cb1kenobi** [comment]
FYI, you shouldn't touch things in the `.vuepress` directory. It gets regenerated when the site is deployed.

**Pull request merged**